src/adapter.rs

```diff
@@ -1033,7 +1079,182 @@ impl Adapter {
         );
     }
 
-    pub fn render_adapter(&self, frame: &mut Frame, color_mode: ColorMode) {
+    pub fn render_status_bar(&self, frame: &mut Frame, status_bar_block: Rect) {
+        let status_bar_content = Paragraph::new("Q: quit Arrows/HJKL: move Space: connect ?: help")
+            .style(
+                Style::default()
+                    .fg(match self.config.color_mode {
+                        ColorMode::Light => Color::White,
+                        _ => Color::Black,
+                    })
+                    .bg(match self.config.color_mode {
+                        ColorMode::Light => Color::Black,
+                        _ => Color::White,
+                    }),
+            );
+
+        frame.render_widget(status_bar_content, status_bar_block);
+    }
+
+    pub fn render_station_mode(&self, frame: &mut Frame, focused_block: FocusedBlock) {
+        if frame.size().height > 30 {
```

Comment:
Can we add that into the config as well ? 
The value need to be taken into account for a better display for popups (help, auth, etc)